src/back/services/vote.ts

```diff
@@ -14,20 +13,37 @@ export class VoteService {
 
   static async getTopVoters(start: Date, end: Date, limit = DEFAULT_TOP_VOTERS_LIMIT) {
     const votes = await SnapshotService.getAllVotesBetweenDates(new Date(start), new Date(end))
-    return this.countVotesByUser(votes)
-      .sort((a, b) => b['votes'] - a['votes'])
-      .slice(0, limit)
+    return this.getSortedVoteCountPerUser(votes).slice(0, limit)
   }
 
-  private static countVotesByUser(votes: SnapshotVote[]) {
+  public static getSortedVoteCountPerUser(votes: SnapshotVote[]) {
     const votesByUser = votes
-      .filter((vote) => vote.vp && vote.vp > VOTES_VP_THRESHOLD)
+      .filter((vote) => vote.vp && vote.vp > 5)
```

Comment:
I'd leave the VOTES_VP_THRESHOLD variable here. As it makes more clear the intention of this filter.